util/config/gather-cray-prgenv-arguments

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+
+THIS_DIR=`dirname "$0"`
+CHPL_HOME="$THIS_DIR/../../"
+
+COMMANDS=`cc -craype-verbose 2>/dev/null`
+
+COMPILE=0
+LINK=0
+case $1 in
+  compile)
+    COMPILE=1
+    ;;
+  link)
+    LINK=1
+    ;;
+  *)
+    echo "Usage: script [compile|link]"
+    ;;
+esac
+
+if [ $? -eq 0 ]; then
+
+  for arg in $COMMANDS
+  do
+    if [[ $arg == -I* ]]
+    then
+      if [[ $COMPILE == 1 ]]; then
```

Comment:
I think all of these can be collapsed into a single if stmt with the `&&` operator. I think it would be slightly more readable, too. E.g.

```bash
if [[ $arg == -I* && $COMPILE == 1 ]] ; then
    echo $arg
fi
```

I think that's the advantage of using the double brackets. With single brackets, the `-a` flag has to be used for logical and, I believe.